### PR TITLE
Add cats and some base objects

### PR DESCRIPTION
**Pull request by adrhill**

### CODE REVIEW

File: src/WatchJuliaBurn.jl [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
+const ➡️😻 = hcat
+const 😹 = cat
+const ⬇️😹 = vcat
+const ➡️😹 = hcat
+const 🐈 = cat
+const ⬇️🐈 = vcat
+const ➡️🐈 = hcat
+
+## Math
+const ㅠ = π
```
Comment (theogf) [inline src/WatchJuliaBurn.jl:31]: I am not sure this one is really needed :laughing:

**Review by theogf — COMMENTED** [review]
Can you also add everything to the README table?

**Comment by theogf** [comment]
Also, tests?

File: src/WatchJuliaBurn.jl (hunk 1 above)
Comment (theogf) [reply src/WatchJuliaBurn.jl:31]: But why not!

Comment (adrhill) [reply src/WatchJuliaBurn.jl:31]: Exactly!

File: README.md [hunk 2]
```diff
@@ -8,3 +8,9 @@ Don't hesitates to add yours via PR.
 | Emoji | Function/Value |
 | --- | --- |
 | c╯°□°ↄ╯ | `throw` |
+|  ✅, 👍, 👌 | `true` |
+|  👎 | `false` |
+| 😻, ⬇️😻, ➡️😻, 😹, ⬇️😹, ➡️😹, 🐈, ⬇️🐈, ➡️🐈 | `cat` |
```
Comment (theogf) [inline README.md]: Is that correct? You used the same emojis for `cat` and `vcat/hcat`

Comment (adrhill) [reply README.md]: Nope, fixed it!

**Review by theogf — APPROVED** [review]

**Pull request merged**